### PR TITLE
Small release helper fixes

### PR DESCRIPTION
**Pull request by CoderDake**

### CODE REVIEW

File: tool/lib/commands/release_helper.dart [hunk 1]
```diff
@@ -86,11 +86,11 @@ class ReleaseHelperCommand extends Command {
 
       final getNewVersionResult = await processManager.runProcess(
         CliCommand.tool(
-          ['update-version', 'current-version', '|', 'tail', '-1'],
+          ['update-version', 'current-version'],
         ),
       );
 
-      final newVersion = getNewVersionResult.stdout.trim();
+      final newVersion = getNewVersionResult.stdout.split('\n').last.trim();
```
Comment (kenzieschmoll) [inline tool/lib/commands/release_helper.dart:93]: curious: why is this change necessary?

Comment (CoderDake) [reply tool/lib/commands/release_helper.dart:93]: I don't think the pipe works in the run process command. It was failing when I ran.
So instead of ping to tail, I'm just grabbing the last line on our side.
